idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -169,7 +169,7 @@ def subsample_single_fasta(input_files_basename, target_n_reads, randgen=random.
     write_to_log("target reads: %d" % target_n_reads)
     if total_records <= target_n_reads:
         return input_file
-    records_to_keep = set(randgen.sample(xrange(total_records), target_n_reads))
```

Comment:
same comment here, let's make the change from xrange to range once we actually switch to python3. Subsample code is already pretty heavy-weight.